qemu/tests/netperf_stress.py

```diff
@@ -23,6 +24,12 @@ def run(test, params, env):
     :param env: Dictionary with test environment.
     """
     def netperf_test(duration):
+        smp = multiprocessing.cpu_count()
```

Comment:
Here I think you want to check the guest CPU instead of host. And when smp > 32, limit it to 32 as @vivianQizhu suggested.

So before start the VM, make `params['smp'] = params['vcpu_maxcpus'] = min(smp, 32)`